camel/agents/chat_agent.py

```diff
@@ -165,6 +146,7 @@ def reset(self):
         """
         self.terminated = False
         self.init_messages()
+        self.token_limit_terminator.reset()
```

Comment:
Why aren't self.response_terminators reset?

Reply:
If they must be reset, then they become mutated actor arguments that is very bad. What if the same terminator is passed to several different agents?

Reply:
For the the terminators passed to the chat agent is agent specific. I will add more information for this in the docs.

Reply:
Will also reset the `response_terminators`